app/src/main/java/org/kiwix/kiwixmobile/settings/SliderPreference.java

```diff
@@ -33,6 +33,8 @@
 
   protected int mSeekBarValue;
 
+  protected int mInitialSeekBarValue;
```

Comment:
We don't use Hungarian notations you should change it to `initialSeekBarValue`. Refractor wherever you have used Hungarian notation.